python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -1,17 +1,3 @@
-# ---
-# jupyter:
-#   jupytext:
-#     text_representation:
-#       extension: .py
-#       format_name: percent
-#       format_version: '1.3'
-#       jupytext_version: 1.11.3
-#   kernelspec:
-#     display_name: Python 3 (ipykernel)
-#     language: python
-#     name: python3
-# ---
```

Comment:
Isn't this still required to generate the empty notebooks with jupyter via the `Makefile`?

Reply:
I didn't know that! thanks for the catch!